src/main/resources/filing/templates/small-full-accounts.html

```diff
@@ -2027,8 +2027,13 @@ <h2 class="text--center">Balance sheet</h2>
 
                       <tr class="figures">
                           <td id="stocks-label">Stocks:</td>
-                          <td class="figure noteIndex" id="uk-busTotalInventoriesCY_ENDNoteIndexes"></td>
-
+                          
+                          {{ if .small_full_accounts.balance_sheet_notes.stocks }}  
+                              <td class="figure noteIndex" id="uk-busTotalInventoriesCY_ENDNoteIndexes">{{getValue "stocks-notes-id"}}</td>
```

Comment:
I think this id="uk-busTotalInventoriesCY_ENDNoteIndexes should be the `name` instead of `id`
I was expecting to see anid like: `something-like-this-id`. i noticed there are other instances where this happens

Reply:
I spoke with @JMetherell and we are leaving it as it is since it matches the template. 
If this is wrong,  it will be fixed in a future PR